MFVNeutralino/test/utilities.py

```diff
@@ -5,24 +5,27 @@
 ####
 
 _leptonpresel = bool_from_argv('leptonpresel')
-_btagpresel = bool_from_argv('btagpresel')
-_presel_s = '_leptonpresel' if _leptonpresel else '_btagpresel' if _btagpresel else ''
+_metpresel = bool_from_argv('metpresel')
+_presel_s = '_leptonpresel' if _leptonpresel else '_metpresel' if _metpresel else ''
```

Comment:
As with elsewhere, we should expand this to become more general / restore the btag trigger stuff.